Cargo.toml

```diff
@@ -23,6 +23,8 @@ rust-version = "1.65.0"
 default = ["tracing-log", "metrics"]
 # Enables support for exporting OpenTelemetry metrics
 metrics = ["opentelemetry/metrics","opentelemetry_sdk/metrics", "smallvec"]
+# Enables support for unstable OpenTelemetry metric instruments
+otel_unstable = ["opentelemetry/otel_unstable"]
```

Comment:
Exporting `otel_unstable` in `tracing-opentelemetry` seems a bit odd to me. 

What do you think about renaming it to something more descriptive like `metrics_gauge_unstable`? This way, we can remove it once `gauge` stabilizes without revealing its origin from `opentelemetry/otel_unstable`.

Reply:
yes, sounds reasonable

Reply:
[here](https://github.com/tokio-rs/tracing-opentelemetry/pull/129/commits/e4cc5f71fcb9857c5d5530d257458b5c2b78ed20)